taxi/views.py

```diff
@@ -61,10 +63,33 @@ class CarListView(LoginRequiredMixin, generic.ListView):
 class CarDetailView(LoginRequiredMixin, generic.DetailView):
     model = Car
 
+    def get(self, request, *args, **kwargs):
+        car = self.get_object()
+        drivers = car.drivers.all()
+        context = {"car": car, "drivers": drivers}
```

Comment:
Why you need to add drivers into context? You can do into template
car.drivers.all